Dockerfile

```diff
@@ -47,20 +47,12 @@ RUN apt-get update -qq \
     git \
     gnupg \
     locales \
+    nodejs \
```

Comment:
It seems this is v18, which is a bit surprising to me - i would have expected v20.

Due to this, the version is downgraded from v20 to v18

https://packages.ubuntu.com/noble/nodejs

Reply:
aaaah, fun :)

in that case i'll abandon ship :)